sdk/appconfiguration/app-configuration/samples/react-sample/src/components/Page.tsx

```diff
@@ -0,0 +1,160 @@
+/*
```

Comment:
Heads up that some of the files, especially those that `create-react-app` generated, are missing this header

Reply:
Added for some more